Packs/CortexXDR/Playbooks/playbook-Calculate_Severity_-_Cortex_XDR_Risky_Assets.yml

```diff
@@ -0,0 +1,702 @@
+id: Calculate Severity - Cortex XDR Risky Assets
+version: -1
+name: Calculate Severity - Cortex XDR Risky Assets
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 8d8663fb-4ab5-4361-81d5-dc1b7ab9892d
+    type: start
+    task:
+      id: 8d8663fb-4ab5-4361-81d5-dc1b7ab9892d
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "11"
+      - "3"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 50
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "1":
+    id: "1"
+    taskid: f0e37a36-7a05-4657-8ff1-fe267d847e96
+    type: title
+    task:
+      id: f0e37a36-7a05-4657-8ff1-fe267d847e96
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 820
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: 4567ad72-940b-4cd1-896a-28b84d9e40e4
+    type: condition
+    task:
+      id: 4567ad72-940b-4cd1-896a-28b84d9e40e4
+      version: -1
+      name: Is there a risky user?
+      description: Checks if there is a risky user, as outputted by the "xdr-list-risky-users" command.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "13"
+      "yes":
+      - "12"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: inputs.XDRRiskyUsers
+                accessor: risk_level
+            iscontext: true
+          right:
+            value: {}
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: inputs.XDRRiskyUsers.score
+                filters:
+                - - operator: isNotEqualString
+                    left:
+                      value:
+                        simple: inputs.XDRRiskyUsers.score
```

Comment:
This is related to the issue we discussed before. The problem stems from the fact that we consider **low risk users** the same as **no risk users** in the risk level output. Our only way to differentiate between the 2 is the score.

Consider this scenario which we're trying to avoid here:
* Incident has "unknown" severity (no reputation on any of the indicators)
* A **non-risky** user is involved, which means he gets a risk level of LOW and a score of 0
* Incident severity rises from "unknown" to **"Low"**, because we considered a non-risky user as a risky user.


Regarding the reason that we check for both: we want to ensure that there is any risk level (it could be changed to ID, score or anything else), **as well as** make sure the score is not 0 (because if it is, user is not risky at all).

Please let me know your thoughts

Reply:
For reference - we agreed to refer only to the risk level for severity determination after weighing the cons and pros